fun.pl

```diff
@@ -4,22 +4,39 @@
           [
           ]).
 
+:- use_module(library(http/http_open)).
 :- use_module(discuss).
 :- use_module(utils).
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% communication predicates
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-fun_answer(List, _, "fortune <word>: furtune message") :-
+fun_answer(List, _, "use <module> help. Available modules: fortune, weather") :-
     member("fun", List),
     member("help", List).
 
+fun_answer(List, _, "fortune <word>: fortune message") :-
```

Comment:
could you just put the help on fun help? That's supposed to be the only help for the module.